apps/dokploy/templates/huly/docker-compose.yml

```diff
@@ -0,0 +1,176 @@
+      - PUID=1000
+      - PGID=1000
+    volumes:
+      - db:/data/db
+    ports:
+      - 27017:27017
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  minio:
+    image: "minio/minio"
+    command: server /data --address ":9000" --console-address ":9001"
+    ports:
+      - 9000:9000
+      - 9001:9001
+    volumes:
+      - files:/data
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  elastic:
+    image: "elasticsearch:7.14.2"
+    command: |
+      /bin/sh -c "./bin/elasticsearch-plugin list | grep -q ingest-attachment || yes | ./bin/elasticsearch-plugin install --silent ingest-attachment;
+      /usr/local/bin/docker-entrypoint.sh eswrapper"
+    volumes:
+      - elastic:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+    environment:
+      - ELASTICSEARCH_PORT_NUMBER=9200
+      - BITNAMI_DEBUG=true
+      - discovery.type=single-node
+      - ES_JAVA_OPTS=-Xms1024m -Xmx1024m
+      - http.cors.enabled=true
+      - http.cors.allow-origin=http://localhost:8087
+    healthcheck:
+      interval: 20s
+      retries: 10
+      test: curl -s http://localhost:9200/_cluster/health | grep -vq '"status":"red"'
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  account:
+    image: hardcoreeng/account:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+    ports:
+      - 3001:3001
+    environment:
+      - SERVER_PORT=3001
+      - SERVER_SECRET=${SERVER_SECRET}
+      - MONGO_URL=mongodb://mongodb:27017
+      - TRANSACTOR_URL=ws://transactor:3333;ws://${SERVER_ADDRESS}:3333
+      - MINIO_ENDPOINT=minio
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+      - FRONT_URL=http://front:8080
+      - INIT_WORKSPACE=demo-tracker
+      - MODEL_ENABLED=*
+      - ACCOUNTS_URL=http://${SERVER_ADDRESS}:3001
+      - ACCOUNT_PORT=3001
+    restart: unless-stopped
+    networks:
+      - dokploy-network
+  front:
+    image: hardcoreeng/front:${HULY_VERSION}
+    links:
+      - mongodb
+      - minio
+      - elastic
+      - collaborator
+      - transactor
+    ports:
+      - 8087:8080
+    environment:
+      - SERVER_PORT=8080
+      - SERVER_SECRET=${SERVER_SECRET}
+      - ACCOUNTS_URL=http://${SERVER_ADDRESS}:3001
+      - REKONI_URL=http://${SERVER_ADDRESS}:4004
+      - CALENDAR_URL=http://${SERVER_ADDRESS}:8095
+      - GMAIL_URL=http://${SERVER_ADDRESS}:8088
+      - TELEGRAM_URL=http://${SERVER_ADDRESS}:8086
+      - UPLOAD_URL=/files
+      - ELASTIC_URL=http://elastic:9200
+      - COLLABORATOR_URL=ws://${SERVER_ADDRESS}:3078
+      - COLLABORATOR_API_URL=http://${SERVER_ADDRESS}:3078
+      - MINIO_ENDPOINT=minio
+      - MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY}
+      - MINIO_SECRET_KEY=${MINIO_SECRET_KEY}
+      - MONGO_URL=mongodb://mongodb:27017
+      - TITLE=Huly Self Hosted
+      - DEFAULT_LANGUAGE=en
+      - LAST_NAME_FIRST=true
+    restart: unless-stopped
+    networks:
+      - dokploy-network
```

Comment:
is not necessary to add the dokploy network, when you assign a domain to a service it automatically will add this network